### ISSUE TITLE
geowombat v1.11.4  breaks conda

### ISSUE THREAD

**Issue by mmann1123** [jgrss/geowombat#188]
Just got this report back from the [conda feedstock](https://github.com/conda-forge/geowombat-feedstock/pull/39) or for details [go here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=561125&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=841356e0-85bb-57d8-dbbc-852e683d1642&l=907)

Assuming pinning the xarray version pulls the older rasterio - breaking things downstream?  What do you think @jgrss 

```
The reported errors are:
- Encountered problems while solving:
-   - package rasterio-1.0.23-py27h5b3f9e8_0 requires libgdal >=2.4.1,<2.5.0a0, but none of the providers can be installed
-   - package pyproj-1.9.4-py27_0 requires python 2.7*, but none of the providers can be installed
```

**Comment by jgrss** [comment]
Hmm, `geowombat==1.11.4` pulls a [higher version of rasterio](https://github.com/jgrss/geowombat/blob/a6c7f31780ef38218344dec3c2cfe41ac1a8a6f4/setup.cfg#L37) than `v1.0.23`. I just did a fresh installation from `main` and it installed `rasterio==1.3.2`. Is `conda` free to build any version of `rasterio` in the build? I don't see a version requirement for it.

**Comment by mmann1123** [comment]
Yeah I don't pin any versions in the [feedstock](https://github.com/conda-forge/geowombat-feedstock/blob/main/recipe/meta.yaml) and it should be able to pull whatever solves all requirements.  So not sure... Instead of pinning xarray could we just implement the np.max min as suggested in  #186?  (although that seemed to open other issues possibly)

**Comment by jgrss** [comment]
Can you see if this is still an issue after #189?

**Comment by mmann1123** [comment]
Ok looks like it builds 2.0 fine on everything but python 3.7. regardless of OS. Should we pin python >=3.8

Looks like opencv is the issue. 
-   - nothing provides libopencv 4.2.0 py36_5 needed

**Comment by mmann1123** [comment]
@jgrss I think finally step is to pin python 3.8 or greater

**Comment by jgrss** [comment]
See #200

**Comment by jgrss** [comment]
#200 is merged into `main` so let me know how the `conda` build goes.

**Comment by mmann1123** [comment]
After much futzing I figured out that adding the `skip` line to meta.yml in the feedstock would allow us to avoid building in py3.7. Should be merging soon.

```
build:
  number: 0
  skip: true  # [py<38]
  script: {{ PYTHON }} setup.py build && pip install .  
```

**Comment by mmann1123** [comment]
I just approved the python version pin. I will let you squash it

**Comment by jgrss** [comment]
This is closed from ~month ago. Is this correct?

**Comment by mmann1123** [comment]
Yep this is resolved
